ocaml/tests/test_observer.ml

```diff
@@ -514,6 +596,7 @@ let test =
     , `Quick
     , test_observer_create_and_destroy
     )
+  ; ("test_most_frequent_spans", `Quick, test_most_frequent_spans)
```

Comment:
If I put this as a later test, it fails. The issue being the tables are not empty at the beginning of the test.

Reply:
That means they are not unit tests currently. Tests should be independent of each other whenever possible